openedx/core/djangoapps/programs/utils.py

```diff
@@ -351,4 +351,12 @@ def supplement_program_data(program_data, user):
             # TODO: Currently unavailable on LMS.
             run_mode['marketing_url'] = ''
 
+            certificate_data = certificate_api.certificate_downloadable_status(user, course_key)
+            certificate_uuid = certificate_data.get('uuid')
+            if certificate_uuid:
```

Comment:
If there's no certificate_uuid should we build the alternate version of the URL with the user and the course?  (I'm not sure, myself)